src/Components/Comment.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import DeleteIcon from '@mui/icons-material/Delete';
+import SortByAlphaIcon from '@mui/icons-material/SortByAlpha';
+
+export class Comment extends Component {
+  render() {
+    const comments = this.props.comments;
```

Comment:
don't create variable in render , as this.props.comments will again and again assign into comments variable,
directly use this.props.comments in place of comments variable